runtime/src/lib.rs

```diff
@@ -345,9 +347,11 @@ impl stake::Config for Runtime {
 	type BondDuration = BondDuration;
 	type MaxValidators = MaxValidators;
 	type MaxNominatorsPerValidator = MaxNominatorsPerValidator;
+	type MaxValidatorsPerNominator = MaxValidatorsPerNominator;
 	type IssuancePerRound = IssuancePerRound;
 	type MaxFee = MaxFee;
 	type MinValidatorStk = MinValidatorStk;
+	type MinNomination = MinNominatorStk;
```

Comment:
This sets the minimum nomination as equal to `MinNominatorStk = 5 * GLMR`, but it could be set at some value less than `MinNominatorStk`. These limits exist to prevent trivial nominations from abusing runtime storage.